tests/specs/kontrol/test-owneruponlytest-testincrementasowner-0-spec.k

```diff
@@ -5,9 +5,7 @@ module TEST-OWNERUPONLYTEST-TESTINCREMENTASOWNER-0-SPEC
 
     claim [BASIC-BLOCK-1-TO-3]: <kevm>
            <k>
-             ( .K => #end EVMC_SUCCESS
```

Comment:
These kontrol specs are now failing, not sure yet what changed this, as the changes in this PR only work with the `CANCUN` schedule, and these specs still use `SHANGHAI`.